authorisations-api.apibp

```diff
@@ -94,23 +101,29 @@ Upon completion the payment will be in one of the following [states](https://doc
 | `error` | An unexpected error occurred either with the underlying PSP or with GOV.UK Pay. No money is taken. |
 
 
-
 + Request (application/json)
     
    
     + Attributes
-        + `continuation_token`: `123abc123` (string, required) - the continuation token provided in the `auth_url_post` link member of the create payment API response
-        + `card_number`: `4242424242424242` (string) - 12-19 digits
-        + `cvc`: `123` (string) - 3-4 digits
-        + `expiry_date`: `02/09` (string) - 5 character string in MM/YY format
-        + `cardholder_name`: `Joanne Smith` (string, optional)
+        + `one_time_token`: `123abc123` (string, required) - the one time token provided in the `auth_url_post` link member of the create payment API response
+        + `card_number`: `4242424242424242` (string, required) - 12-19 digits
+        + `cvc`: `123` (string, required) - 3-4 digits
+        + `expiry_date`: `02/09` (string, required) - 5 character string in MM/YY format
+        + `cardholder_name`: `Joanne Smith` (string, required)
         
-+ Response 201 (application/json)
++ Response 204 (application/json)
 
-        Created
++ Response 400 (application/json)
```

Comment:
Would it be good to add the 422 error code, or do we think that is implied by the rest of the API docs?